gateway/src/test/java/io/camunda/zeebe/gateway/grpc/GrpcErrorMapperTenantTest.java

```diff
@@ -97,14 +95,20 @@ void shouldNotLogInvalidTenantRequestException(final String invalidTenantId) {
 
   public static Stream<Arguments> invalidTenantIds() {
     return Stream.of(
-        Arguments.of("tenant!@#", "tenant identifier contains illegal characters"),
-        Arguments.of("", "no tenant identifier was provided"),
-        Arguments.of("     ", "no tenant identifier was provided"),
-        Arguments.of("a".repeat(35), "tenant identifier is longer than 31 characters"),
-        Arguments.of("abcde.-  ", "tenant identifier contains illegal characters"));
+        Arguments.of("tenant!@#", true, "tenant identifier contains illegal characters"),
+        Arguments.of("", true, "no tenant identifier was provided"),
+        Arguments.of("     ", true, "no tenant identifier was provided"),
+        Arguments.of("a".repeat(35), true, "tenant identifier is longer than 31 characters"),
+        Arguments.of("abcde.-  ", true, "tenant identifier contains illegal characters"));
   }
 
-  public static List<String> validTenantIds() {
-    return List.of("tenant1", "tenant-2", "tenant.3", "tenant.test-5");
+  public static Stream<Arguments> validTenantIds() {
+    return Stream.of(
+        Arguments.of("tenant1", true),
+        Arguments.of("tenant-2", true),
+        Arguments.of("tenant.3", true),
+        Arguments.of("tenant.test-5", true),
+        Arguments.of("<default>", false),
```

Comment:
💡 I have one last idea from these test cases

🔧 Let's add a valid tenant id case for `<default>` when multi-tenancy is enabled.
🔧 Let's add an invalid tenant id cases like `tenant1` when multi-tenancy is disabled.

Reply:
That was a nice suggestion @korthout, I had a bug in my tenant id validation logic. Have a look now.